api/policies/sessionAuth.js

```diff
@@ -8,7 +8,7 @@ var fail = function (res) {
 }
 
 module.exports = function (req, res, next) {
-  if (TokenAuth.isAuthenticated(res)) {
+  if (AccessTokenAuth.isAuthenticated(req)) {
```

Comment:
this if-clause can be dropped entirely; since `req.session.userId` is set now, the `UserSession.isLoggedIn` check will be true